setup.py

```diff
@@ -45,10 +35,10 @@
 with open(README_PATH) as readme_file:
     README = readme_file.read()
 
-
 setuptools.setup(
-    name="qiskit-aqt-provider",
+    name="qiskit_aqt_provider",
```

Comment:
This should be left alone right? The package name is qiskit-aqt-provider still. The project namespace is qiskit_aqt_provider.

Reply:
ok yeah sorry